zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -212,7 +212,8 @@ ensemble:
   consists of a single line containing only the text of that machine's
   id. So *myid* of server 1 would contain the text
   "1" and nothing else. The id must be unique within the
-  ensemble and should have a value between 1 and 255.
+  ensemble and must have a value between 0 and 255.
```

Comment:
Is there a reason why we are changing the official spec here? (as official as we get at least). Perhaps we should continue to enforce the "don't use zero" rule? Not sure I have a strong feeling either way but this has been on the books since ZOOKEEPER-260

Reserving 0 may be a good idea for something in the future...?